pkg/cmd/cli/describe/printer.go

```diff
@@ -427,7 +428,9 @@ func printOAuthClientAuthorizationList(list *oauthapi.OAuthClientAuthorizationLi
 }
 
 func printOAuthAccessToken(token *oauthapi.OAuthAccessToken, w io.Writer) error {
-	_, err := fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%d\t%s\t%s\n", token.Name, token.UserName, token.ClientName, token.CreationTimestamp, token.ExpiresIn, token.RedirectURI, strings.Join(token.Scopes, ","))
+	created := token.CreationTimestamp
```

Comment:
Storing this and the immediately using the original read weird.  Not that I doubt their equivalence, but you want to make up your mind one way or the other?